packages/inference/README.md

```diff
@@ -68,6 +68,7 @@ Only a subset of models are supported when requesting third-party providers. You
 - [Replicate supported models](./src/providers/replicate.ts)
 - [Sambanova supported models](./src/providers/sambanova.ts)
 - [Together supported models](./src/providers/together.ts)
+- [Fireworks AI supported models](./src/providers/fireworks-ai.ts)
```

Comment:
The links in this list are not correct anymore. Must be a link to the API route now:

```
- [Fal.ai supported models](https://huggingface.co/api/partners/fal-ai/models)
- [Fireworks AI supported models](https://huggingface.co/api/partners/fireworks-ai/models)
- [Replicate supported models](https://huggingface.co/api/partners/replicate/models)
- [Sambanova supported models](https://huggingface.co/api/partners/sambanova/models)
- [Together supported models](https://huggingface.co/api/partners/together/models)
- [HF Inference API (serverless)](https://huggingface.co/models?inference=warm&sort=trending)
```

(+ I would order alphabetically and put HF Inference in last position)